fabtests/common/hmem.c

```diff
@@ -43,7 +43,7 @@ struct ft_hmem_ops {
 	int (*alloc_host)(void **buf, size_t size);
 	int (*free)(void *buf);
 	int (*free_host)(void *buf);
-	int (*memset)(uint64_t device, void *buf, int value, size_t size);
+	int (*set)(uint64_t device, void *buf, int value, size_t size);
```

Comment:
Can we call this ```mem_set``` instead, to make it clear that the behavior is the same as memset?  'set' isn't as obvious to me.